packages/shared-components/src/components/onto-layout/onto-layout.tsx

```diff
@@ -52,6 +54,12 @@ export class OntoLayout {
     this.windowResizeObserver();
   }
 
+  @Listen('single-spa:routing-event',  {target: 'window'})
```

Comment:
I prefer single-spa stuff to be handled in root-config and not to be propagated in the other modules

Reply:
Refactored